src/components/base/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import Theme from '../../theme';
+
+interface Props {
+    color?: string;
+    radius?: string;
+    width?: string;
+    height?: string;
+    margin?: string;
+
+    theme?: Theme;
+}
+
+export default styled.button`
```

Comment:
Missing mouse hover states: Design did not specify how it should look, for now just use an alternative color or lighter shade